server.js

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+// import bodyParser from 'body-parser';
```

Comment:
@chokonaira Please remove dead code from your work so as not to add unnecessary and unneeded code to your codebase